release-notes/opensearch-security.release-notes-2.7.0.0.md

```diff
@@ -0,0 +1,40 @@
+## 2023-04-25 Version 2.7.0.0
+
+Compatible with OpenSearch 2.7.0
+
+### Bug Fixes
+
+* Support multitenancy for the anonymous user ([#2459](https://github.com/opensearch-project/security/pull/2459))
+* Fix error message when system index is blocked ([#2525](https://github.com/opensearch-project/security/pull/2525))
+* Fix of OpenSSLTest is not using the OpenSSL Provider ([#2301](https://github.com/opensearch-project/security/pull/2301))
+* Add chmod 0600 to install_demo_configuration bash script ([#2550](https://github.com/opensearch-project/security/pull/2550))
+* Fix SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder" ([#2564](https://github.com/opensearch-project/security/pull/2564))
+* Fix lost privileges during auto initializing of the index ([#2498](https://github.com/opensearch-project/security/pull/2498))
+* Fix NPE and add additional graceful error handling ([#2687](https://github.com/opensearch-project/security/pull/2687))
+
+### Enhancements
+
+* Flatten response times ([#2471](https://github.com/opensearch-project/security/pull/2471))
```

Comment:
This change was included in 2.6.0, but it looks like it was not added in the release notes. Can you take it out of this file and add it into the release note for 2.6?

Reply:
Good catch! Done